### ISSUE TITLE
Cannot read property 'then' of undefined for GoogleTagManager pushDataLayerEvent

### ISSUE THREAD

**Issue by codetony25** [idehub/react-native-google-analytics-bridge#99]
So i've been able to implement the Google Analytics part and I am able to see the results successfully. However now that I am trying to implement GoogleTagManager I get that `.then` is undefined when trying to log out a success message.

Here's my implementation:

```js
import GoogleAnalytics, {
  GoogleAnalyticsTracker,
  GoogleTagManager,
  GoogleAnalyticsSettings
} from 'react-native-google-analytics-bridge';

class AnalyticState {

  constructor() {
    GoogleTagManager.openContainerWithId('GTM-ASDFGHJKL')
      .then(() => GoogleTagManager.stringForKey('pack'))
      .then((str) => console.log('Pack: ', str));

    GoogleTagManager.pushDataLayerEvent({
        event: 'openScreen',
        screenName: 'Home Screen',
        appName: 'MyAwesomeApp'
      }).then((success) => { console.log('GTM Success', success) });
  }

}

export default new AnalyticState();
```

Right now in the console I am getting the following messages:

- `Pack:  `
- `Cannot read property 'then' of undefined`
- `Unhandled JS Exception: Cannot read property 'then' of undefined`

I just included the GoogleTagManager part, but let me know if you guys want to see the rest, I can definately do so.

Thank you in advance!

**Comment by sheldonj** [comment]
Hey there. I was having the same issue. I submitted a pull request for the the fix, but it hasn't been merged yet. #96

**Comment by codetony25** [comment]
@sheldonj 's solution works! Please merge this! :) I'll close this issue once that's done

**Comment by cbrevik** [comment]
Released the fix with version `4.0.3` of this module. Please verify that it is fixed, and close issue if that is the case :+1:

**Comment by codetony25** [comment]
This is fixed with the latest update 4.0.3, Thank you! closing.
